<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWidget</class>
 <widget class="QWidget" name="MyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyWidget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>set boy age:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>79</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>set girl age:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxBoy">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxGirl">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBoyInc">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>boy age+1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGirlInc">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>girl age+1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClassInfo">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>124</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Meta-Object info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>361</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
